### ISSUE TITLE
Possible bug in obspy.taup

### ISSUE THREAD

**Issue by jsaul** [obspy/obspy#2271]
Hello all,

while attempting to use obspy.taup to validate traveltime tables, to have a 2nd opinion, so to speak, I ran into a few issues.

See this minimal example. It computes a few regional phases for source at 10 km depth and a station at 3 degrees distance:
```
import obspy.taup

taupmodel = obspy.taup.TauPyModel(model="iasp91")

pi_over_180 = 0.017453292519943295

for arr in taupmodel.get_travel_times(source_depth_in_km=10, distance_in_degree=3, phase_list=["Pn","Pg","Pb"]):
    print("%-12s %8.3f %8.5f %8.5f" % (arr.name, arr.time, arr.ray_param*pi_over_180, arr.takeoff_angle))
```

The output is:
> Error with this phase, skipping it: Pb
> Pn             47.581 13.75424 45.93554
> Pg             53.503 17.04925 62.96108
> Pg             57.639 19.05667 84.60560

The numbers are time, angular slowness and takeoff angle.

There are two Pg phases computed, one with a relatively "horizontal" takeoff angle, another with a much steeper takeoff angle. For comparison this is the same table generated with libtau as used by SeisComP:

> Pn             47.579 13.75078 45.92068
> Pb             53.503 17.04925 62.96109
> Pg             57.489 19.14130 90.22696

Pn is fine. The first Pg arrival in the ObsPy-generated table is practically identical with the Pb of libtau/SeisComP. Also the takeoff angle makes a lot of sense for Pb. So ObsPy seems to have confused these phases. Regarding the Pg arrival at 57+ seconds, hmm. I would expect the takeoff angle not to be more than 5 degrees off from horizontal as in the ObsPy example. The libtau/SeisComP intuitively makes more sense.

For comparison, a source at 1 km depth, same distance.

ObsPy:
> Error with this phase, skipping it: Pb
> Pn             48.661 13.75424 45.85196
> Pg             54.211 17.04974 62.80612
> Pg             57.504 19.16338 88.67172
> Pg             57.813 19.00908 82.60468

libtau/SeisComP:
> Pn             48.659 13.75103 45.83818
> Pb             54.210 17.04974 62.80612
> Pg             57.504 19.16331 88.66305

Again the problem with an apparently mislabelled but otherwise correctly computed Pb. Also the difference of Pn is nothing to be worried about. We now see one of three Pg phases in the ObsPy matching the libtau/SeisComP Pg very well. Almost certainly the same geometric arrival. But what about the additional Pg in the ObsPy list at 57.813? There shouldn't be any.

Tested with current master.

To sum up:
* wrongly labelled Pb phase
* Pg not fully consistent with libtau and "intuition"
* additional Pg phase in one of the examples makes no sense

Let me know if I missed something.

**Comment by jsaul** [comment]
Answering myself... Pb is not supported as own phase, it goes under Pg in the original TauP Toolkit and therefore here, too. Which actually makes a lot of sense from a practical point of view.

Sorry for the noise. ;)

**Comment by megies** [comment]
Good to hear you were able to sort it out!

> Sorry for the noise. ;)

No worries at all. :+1:
